docs/pipelines.md

```diff
@@ -465,6 +465,22 @@ There are a lot of scenarios where `WhenExpressions` can be really useful. Some
 - Checking if the name of a CI job matches
 - Checking if an optional Workspace has been provided
 
+#### Guarding a `Task` and its dependent `Tasks`
+
+When  `WhenExpressions` evaluate to `False`, the `Task` and its branch (of dependent `Tasks`) will be skipped by
+default while the rest of the `Pipeline` will execute. The global default scope of `WhenExpressions` is set to `Branch`;
+`scope-when-expressions-to-task` field in [`config/config-feature-flags.yaml`](./../config/config-feature-flags.yaml)
+defaults to `False`.
```

Comment:
Small nit: suggest `"false"` here to match case with what's already in the configmap & to indicate it's a string value rather than a yaml bool.

Reply:
another nit from me: i think it'd be better to link to https://github.com/tektoncd/pipeline/blob/main/docs/install.md#customizing-the-pipelines-controller-behavior i.e. the doc about the config map vs the config map itself

(ps i think that doc might need an update also)